### PR TITLE
add multiparty direct message support to Slack Web API

### PR DESCRIPTION
**Pull request by shahruz**

### CODE REVIEW

File: lib/Slack_web_api.js [hunk 1]
```diff
@@ -177,6 +177,23 @@ module.exports = function(bot, config) {
                 slack_api.callAPI('im.open',options,cb);
             }
         },
+        mpim: {
+            close: function(options, cb) {
+                slack_api.callAPI('mpim.close',options,cb);
+            },
+            history: function(options, cb) {
+                slack_api.callAPI('mpim.history',options,cb);
```
Comment (anonrig) [inline lib/Slack_web_api.js]: Please use a linter with our .jscsrc file.

**Comment by RafaelCosman** [comment]
Thanks @shahruz! Let me know if I can help you get set up with that.

**Comment by shahruz** [comment]
@RafaelCosman @anonrig Hey, I ran it through the linter and it didn't show any issues. Same with npm test. Am I missing something?

**Comment by anonrig** [comment]
You should add a space after a comma while defining the inputs of a function. If no linter issues are warned, then we should change the jscsrc file accordingly. @shahruz @RafaelCosman

**Comment by shahruz** [comment]
@anonrig @RafaelCosman Got it. I'll put up a separate PR with the jscsrc updated, and the rest of this file updated w/o these new functions.

**Comment by benbrown** [comment]
Can we merge? This is a good patch.

**Comment by anonrig** [comment]
I'm merging this.

**Pull request merged**